src/CMakeLists.txt

```diff
@@ -63,6 +66,17 @@ if(NOT CMAKE_BUILD_TYPE)
           FORCE
   )
 endif()
+if(CMAKE_CXX_COMPILER_ID STREQUAL "GNU")
+  # Policy that enable INTERPROCEDURAL_OPTIMIZATIOn
+  set(CMAKE_POLICY_DEFAULT_CMP0069 NEW)
+  cmake_policy(SET CMP0069 NEW)
```

Comment:
As you pinged me on Slack, I'll enable this across all Magnum projects... but you still need this settings for other subprojects (assimp, bullet) anyway :)